packages/components/Tabs/src/Tabs.tsx

```diff
@@ -137,8 +187,8 @@ export const Tabs = compose<TabsType>({
   slots: {
     root: View,
     label: Text,
-    container: FocusZone,
-    stack: { slotType: View, filter: filterViewProps },
+    container: Platform.OS !== 'windows' ? FocusZone : React.Fragment,
+    stack: View,
```

Comment:
Curious, why did you remove the filterViewProps?

Reply:
Because I needed the componentRef of the stack to focus on it and the filter removed that prop.

Reply:
super nit: "GH #964" instead of "GH 964" to match convention elsewhere and also make it more obvious it's a bug number